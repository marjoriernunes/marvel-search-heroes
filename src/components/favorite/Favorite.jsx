import Icon from '../../assets/Icons';

const Favorite = (props) => {
    var iconName = 'favorito_02';
    const changeFav = () => {
        props.favValue(true);
        iconName = 'favorito_01'
    }

    return (
        <div onClick={changeFav}>
            <Icon name={iconName} width="20px" height="20px"/>
        </div>
    )
}

export default Favorite;