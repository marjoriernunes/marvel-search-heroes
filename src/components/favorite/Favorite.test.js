import {render} from '@testing-library/react'
import Favorite from './Favorite'

test('renders correctly', () =>{
    const utils = render(<Favorite/>)
    expect(utils.container).toMatchSnapshot()
});