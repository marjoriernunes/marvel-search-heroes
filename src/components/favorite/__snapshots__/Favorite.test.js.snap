// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <svg
      height="20px"
      width="20px"
    >
      <use
        xlink:href="icons.svg#icon-favorito_02"
      />
    </svg>
  </div>
</div>
`;
